### PR TITLE
First step #1712-Created a custom layout for a dialog with a checkbox.

### PR DESCRIPTION
**Pull request by sauravvishal8797**
First step for issue #1712 

Changes: Custom layout for a dialog with checkbox added.

### CODE REVIEW

File: app/src/main/res/layout/dialog_checkbox.xml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+            android:orientation="vertical"
+        >
+          <TextView
+              android:id="@+id/text_dialog_message"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:paddingTop="20dp"
+              android:paddingRight="20dp"
+              android:paddingLeft="20dp"
+              android:text="null"
```
Comment (anantprsd5) [inline app/src/main/res/layout/dialog_checkbox.xml]: Not needed.

File: app/src/main/res/layout/dialog_checkbox.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              android:layout_height="wrap_content"
+              android:buttonTint="@color/md_black_1000"
+              android:layout_gravity="center_horizontal"
+              android:layout_weight="1"
+          />
+          <TextView
+              android:id="@+id/checkbox_text_dialog"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:text="null"
```
Comment (anantprsd5) [inline app/src/main/res/layout/dialog_checkbox.xml]: Not required

Comment (sauravvishal8797) [reply app/src/main/res/layout/dialog_checkbox.xml]: @anantprsd5 This textview denotes the text message for the checkbox(which can be anything according to the use of the checkbox).

File: app/src/main/res/layout/dialog_checkbox.xml (hunk 1 above)
Comment (sauravvishal8797) [reply app/src/main/res/layout/dialog_checkbox.xml]: @anantprsd5 This sets up the spacing for the dialog message. Can you please explain why do you think that it's not required here.

File: app/src/main/res/layout/dialog_checkbox.xml (hunk 2 above)
Comment (anantprsd5) [reply app/src/main/res/layout/dialog_checkbox.xml]: @sauravvishal8797 you are setting the text of this textView programmatically so using this line becomes redundant.

Comment (sauravvishal8797) [reply app/src/main/res/layout/dialog_checkbox.xml]: @anantprsd5  Did the changes, please review.

**Pull request merged**